clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -359,10 +396,9 @@ describe("Fides-js TCF", () => {
               { id: PURPOSE_7.id, preference: "opt_in" },
               { id: PURPOSE_9.id, preference: "opt_in" },
             ]);
-            // TODO: fides#4210
```

Comment:
I realized I left all these TODOs which I should've uncommented out in https://github.com/ethyca/fides/pull/4234 but forgot to!